index.html

```diff
@@ -216,55 +216,54 @@ <h2>Speakers</h2>
 		    </div>
 		</div>
 		</div>
-	    <div class="row">
 		<div class="col-md-4">
-		    <div class="speaker_box"><img alt="" src="images/s1.png" title="Speaker">
-			<div class="speak_content"><a href="#" title="Speaker" class="title_head">Keynote Speaker</a>
-			    <h4>Sarah Wilson</h4>
-			    <h5>Software Enfgineer</h5>
+		    <div class="speaker_box"><img alt="" src="images/dave_cheney_profile.png" title="Speaker" width="350px" height="350px">
+			<div class="speak_content"><a href="#" class="title_head">Keynote Speaker</a>
+			    <h4 style="background-color: rgba(0,0,0,0.5);padding: 0.5em; max-width: 300px;">Dave Cheney</h4>
+			    <h5 style="background-color: rgba(0,0,0,0.5);padding: 0.5em; max-width: 300px;">Open Source Contributor</h5>
 			    <div class="social_box">
 				<ul>
-				    <li><a href="https://www.linkedin.com/" target="_blank" class="hvr-wobble-bottom linkedin"
-											    title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
-				    <li><a href="https://www.twitter.com/" target="_blank" class="hvr-wobble-bottom tweet"
+				    <!-- <li><a href="https://dave.cheney.net/" target="_blank" class="hvr-wobble-bottom linkedin"
+											    title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li> -->
+				    <li><a href="https://twitter.com/davecheney" target="_blank" class="hvr-wobble-bottom tweet"
 											   title="Twitter"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
-				    <li><a href="https://www.github.com/" target="_blank" class="hvr-wobble-bottom git"
+				    <li><a href="https://github.com/davecheney" target="_blank" class="hvr-wobble-bottom git"
 											  title="Github"><i class="fa fa-github" aria-hidden="true"></i></a></li>
 				</ul>
 			    </div>
 			</div>
 		    </div>
 		</div>
 		<div class="col-md-4">
-		    <div class="speaker_box"><img alt="" src="images/s2.png" title="Speaker">
-			<div class="speak_content"><a href="#" class="title_head">Keynote Speaker</a>
-			    <h4>Johnathan Doe</h4>
-			    <h5>Software Enfgineer</h5>
+		    <div class="speaker_box"><img alt="" src="images/question_mark_profile.png" title="Speaker" width="350px" height="350px">
+			<div class="speak_content"><a href="#" class="title_head">Speaker</a>
+			    <h4 style="background-color: rgba(0,0,0,0.5);padding: 0.5em;max-width: 300px; ">Maybe you?</h4>
+			    <h5 style="background-color: rgba(0,0,0,0.5);padding: 0.5em;max-width: 300px;">Apply and send your talk</h5>
 			    <div class="social_box">
 				<ul>
-				    <li><a href="https://www.linkedin.com/" target="_blank" class="hvr-wobble-bottom linkedin"
+				    <li><a href="https://www.papercall.io/gcil-2020" target="_blank" class="hvr-wobble-bottom linkedin"
```

Comment:
Maybe a single "submit a talk" button instead of the social ones? They are confusing